app/controllers/integrations/base_controller.rb

```diff
@@ -105,12 +102,6 @@ def service_name
     @service_name ||= self.class.name.demodulize.sub('Controller', '').downcase
   end
 
-  def create_release_and_build_record
-    release = create_new_release
-    release = latest_release unless release && release.persisted?
-    create_build(release.version, [release])
```

Comment:
is there a way to keep this information in the case we create a build after creating a release?